src/index.html

```diff
@@ -10,8 +10,168 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body autocomplete="off">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldset_info">
+        <legend>
+          Personal information
+        </legend>
+        <label class="label_Surname">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            class="marga"
+          >
+        </label>
+        <br>
+        <label class="label_Name">
+        Name:
+          <input
+            type="text"
+            name="Name"
+            class="marga1"
+          >
+        </label>
+        <br>
+        <label class="label_years">
+        How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="old"
+            class="marga2"
+          >
+        </label>
+        <br>
+        <label class="label_birth">
+        Full date of birth:
+          <input
+            type="date"
+            name="date or birth"
+            class="marga3"
+          >
+        </label>
+        <br>
+        <label class="label_agreement">
```

Comment:
Why you need this class if you don't provide relevant css rules? Please delete and check for all cases!!!!